packages/wagmi/src/utils/defaultWagmiReactConfig.ts

```diff
@@ -1,8 +1,8 @@
 import '@web3modal/polyfills'
 
-import type { CreateConfigParameters, CreateConnectorFn, Config } from 'wagmi'
-import { createConfig } from 'wagmi'
-import { coinbaseWallet } from 'wagmi/connectors'
+import type { CreateConfigParameters, CreateConnectorFn, Config } from '@wagmi/core'
+import { createConfig } from '@wagmi/core'
+import { coinbaseWallet } from '@wagmi/connectors'
```

Comment:
is this change needed?

Reply:
I guess, I did this bc of I was getting the wrong dependency when not updated the wagmi version across the project but only for this package and not sure why the import are different for `defaultWagmiCoreConfig.ts` and `defaultWagmiReactConfig.ts`. 
Let me know if this is not needed.